docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -174,6 +199,16 @@ function classrooms_core_preprocess_block(&$variables) {
  */
 function classrooms_core_preprocess_field(&$variables, $hook) {
   switch ($variables['element']['#field_name']) {
+    case 'field_room_type':
+      $link_items = $variables['items'];
+      foreach ($link_items as &$item) {
+        $item['content']['#options']['attributes']['class'][] = 'bttn bttn--primary bttn--light-font bttn--no-text bttn--small';
+        $item['content']['#title'] = Markup::create('<i role="presentation" class="fa-building fas"></i>&nbsp; <span class="field__label">Room Type:</span>&nbsp; ' . $item['content']['#title']);
```

Comment:
Rather than embedding this markup here, should this be a field template?

Reply:
Sure, I'll try that out!